docs/docs/themes/shadowing.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Shadowing
+---
+
+
+Gatsby themes introduce a concept called "shadowing". This feature allows users to override a component, or any other file, in the `src` directory to customize it.
+
+A practical use case is when you've installed `gatsby-theme-blog` and want to customize the author Bio component to add your own biographical content. Component shadowing lets you replace the theme’s original file, `gatsby-theme-blog/src/components/bio.js`, with your own file to override with any changes you need.
+
+## Shadowing Example
+
+If you’ve installed `gatsby-theme-blog` you’ll notice that it renders a `Bio` component which is used in the `BlogPost` template. If you’d like to change the `Bio` component you can do so with component shadowing.
+
+### Theme file structure
+
+You can look at the `gatsby-theme-blog`'s file structure to determine the file path for the file you want to shadow.
+
+```txt
```

Comment:
Add the CLI command that generates this as a title?

```suggestion
```txt:title="tree gatsby-theme-blog"
```